_content/articles/the-sal-an-alternative-buddhist-holy-tree_ireland-John-D.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "The Sal — An Alternative Buddhist Holy Tree?"
+authors:
+  - "John D. Ireland"
+external_url: "https://journal.equinoxpub.com/BSR/article/view/14508/16725"
+source_url: " https://doi.org/10.1558/bsrv.v17i1.14508"
+drive_links:
+  - "https://drive.google.com/drive/folders/10FhXe3dtyCmt_uQt0VXD8hpMJaDCjS2T"
+course: nature
+tags:
```

Comment:
Okay, for tags I'm going to go with:

```yaml
tags:
  - plants
  - setting
```

`trees` is okay but I think this paper is general enough to be worth bumping up a conceptual level to `plants` because any Buddhist interested in plants should enjoy this article, not just those interested in trees specifically.

For India and Nepal at the time of the Buddha, that tag is `setting`. The `india` and `nepal` (lowercase) tags are used for the secular, political histories of those nation states. :)